yt/geometry/tests/test_particle_octree.py

```diff
@@ -175,11 +176,12 @@ def test_bitmap_no_collisions():
     sub_mi2 = np.zeros(max_npart, "uint64")
     posgen = yield_fake_decomp('sliced', npart, nfiles,
                                left_edge, right_edge)
+    coll = None
     for i, (pos, hsml) in enumerate(posgen):
-        nsub_mi = reg._refined_index_data_file(
-            pos, hsml, mask, sub_mi1, sub_mi2, i, 0)
-        reg._set_refined_index_data_file(
-            sub_mi1, sub_mi2, i, nsub_mi)
+        nsub_mi, coll = reg._refined_index_data_file(
+            coll, pos, hsml, mask, sub_mi1, sub_mi2, i,
```

Comment:
Could you use `None` on l182 here like you did at 218? Or am I missing something subtle that coll needs to be defined here on l179 but not there?